src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/content/main.js

```diff
@@ -17,7 +17,12 @@ define(function() {
                 url: '/admin/content-navigations?alias=account'
             },
             toolbar: {
-                template: 'default'
+                buttons: [
+                    'saveWithOptions',
+                    {'settings': {
+                        dropdownItems: ['delete']
+                    }}
```

Comment:
for me thats a strange syntax ... /cc @chirimoya @danrot 

i think it would be better something like this:
```
buttons: [
    {id:'saveWithOptions'},
    {
        id:'settings',
        options: {dropdownItems: ['delete']}
    }
]
```